### DISCUSSION TITLE
Accept/Navigate Text Completion in Editor

### DISCUSSION THREAD

**Discussion by jsbutcher** [ananthakumaran/paisa#118] [Q&A]
I just started using Paisa and am excited about it so far! I'm hoping to learn how to take advantage of the autocomplete feature in the Editor. I can select autocomplete suggestions with the mouse cursor, but none of the traditional keybinds (arrow keys, Tab, End) allow me to either navigate the autocomplete suggestions or to select the highlighted autocomplete option. Help is appreciated. Thanks!

**Comment by jsbutcher** [comment]
For whatever reason, right after I submitted this discussion post, the Enter key and arrow keys began working appropriately in the Editor's autocomplete menu, despite having fiddled with it for a good half-hour last night and this morning. Perhaps it has something to do with my browser's focus or something? I don't know for sure, but I'll mark this as closed, now.

**Reply by ananthakumaran** [reply]
It's possible you hit a hard to reproduce bug. I have noticed a few oddities on mobile views, which I assumed were due to space constraints. See if you can find any way to reproduce the issue reliably, I can then investigate more.
